src/forward_index.c

```diff
@@ -142,17 +142,14 @@ static khIdxEntry *makeEntry(ForwardIndex *idx, const char *s, size_t n, uint32_
   return (khIdxEntry *)bb;
 }
 
-// void ForwardIndex_NormalizeFreq(ForwardIndex *idx, ForwardIndexEntry *e) {
-//   e->freq = e->freq / idx->maxFreq;
-// }
-int forwardIndexTokenFunc(void *ctx, const Token *t) {
-  ForwardIndex *idx = ctx;
-
+static void ForwardIndex_HandleToken(ForwardIndex *idx, const char *tok, size_t tokLen,
```

Comment:
why drop the token?

Reply:
Micro optimization to avoid having to re-read the struct field again.